dnx-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,263 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachines_dnx_on_ubuntu_adminPassword": {
```

Comment:
The parameter names should be cleaned up. These look like exported from the portal